Packs/AWS_SystemManager/Integrations/AWSSystemManager/AWSSystemManager.py

```diff
@@ -0,0 +1,1308 @@
+                "Comment": command.get("Comment"),
+                "Target Count": command.get("TargetCount"),
+                "Error Count": command.get("ErrorCount"),
+                "Delivery Timed Out Count": command.get("DeliveryTimedOutCount"),
+                "Completed Count": command.get("CompletedCount"),
+            }
+            for command in commands]
+
+    kwargs: "ListCommandsRequestRequestTypeDef" = {
+        "MaxResults": arg_to_number(args.get("limit", 50)) or 50,
+    }
+    if next_token := args.get("next_token"):
+        kwargs["NextToken"] = next_token
+    if command_id := args.get("command_id"):
+        kwargs["CommandId"] = command_id
+    response = ssm_client.list_commands(**kwargs)
+    response = convert_datetime_to_iso(response)
+
+    commands = response.get("Commands", [])
+    command_result = []
+    if next_token := response.get("NextToken"):
+        command_result.append(
+            next_token_command_result(next_token, "CommandNextToken"),
+        )
+
+    command_result.append(CommandResults(
+        outputs=commands,
+        outputs_key_field="CommandId",
+        outputs_prefix="AWS.SSM.Command",
+        readable_output=tableToMarkdown(
+            name="AWS SSM Commands",
+            t=_parse_list_command(commands),
+            headers=["Command Id", "Status", "Requested date",
+                     "Document name", "Comment", "Target Count", "Error Count",
+                     "Delivery Timed Out Count", "Completed Count"],
+        ),
+    ))
+    return command_result
+
+
+@polling_function(
+    name="aws-ssm-command-run",
+    interval=arg_to_number(demisto.args().get("interval_in_seconds", 30)),
+    requires_polling_arg=False,  # means it will always be default to poll, poll=true,
+)
+def run_command_command(args: dict[str, Any], ssm_client: "SSMClient") -> PollResult:  # TODO Invocation
+    if command_id := args.get("command_id"):
+        status = get_command_status(command_id, ssm_client)
+        if status in FINAL_STATUSES_COMMAND:
+            return PollResult(
+                response=CommandResults(
+                    readable_output=FINAL_STATUSES_COMMAND[status],
+                ),
+                continue_to_poll=False,
+            )
+        else:
+            return PollResult(
+                partial_result=CommandResults(readable_output=f"Command {command_id} is {status}"),
+                continue_to_poll=True,
+                args_for_next_run=args,
+                response=None,
+            )
+
+    kwargs: "SendCommandRequestRequestTypeDef" = {
+        "DocumentName": args["document_name"],
+        "InstanceIds": argToList(args["instance_ids"]),
+        "DocumentVersion": format_document_version(args.get("document_version")),
+    }
+    if comment := args.get("comment"):
+        kwargs["Comment"] = comment
+    if output_s3_bucket_name := args.get("output_s3_bucket_name"):
+        kwargs["OutputS3BucketName"] = output_s3_bucket_name
```

Comment:
repetitive, consider
```python
def update_if_value(input_dictionary :dict, output_dictionary: dict, input_to_output_keys: dict[str, str]):
    for input_key, output_key in input_to_output_keys.items():
        if value := input_dictionary.get(input_key):
            output_dictionary[output_key] = value
```

This will save a lot of code.

Reply:
Done